Taskfile.yml

```diff
@@ -0,0 +1,33 @@
+version: '3'
+
+dotenv: ['.env']
+
+tasks:
+  run:
+    cmds:
+      - go run main.go
+  dev_server:
```

Comment:
The use of `go run` for tasks like `dev_server` and `run`, which includes starting the server, may not cleanly handle application errors or runtime issues. It's recommended to build the application first and then execute the binary to avoid potential issues during development.